libs/gltfio/src/ResourceLoader.cpp

```diff
@@ -932,36 +781,36 @@ void ResourceLoader::Impl::computeTangents(FFilamentAsset* asset) {
     }
 
     // Create a job description for each primitive.
-    std::vector<JobParams> jobParams;
+    std::vector<TangentsJob::Params> jobParams;
     for (auto pair : asset->mPrimitives) {
         VertexBuffer* vb = pair.second;
         auto iter = baseTangents.find(vb);
         if (iter != baseTangents.end()) {
-            jobParams.emplace_back(JobParams { pair.first, vb, iter->second, kMorphTargetUnused });
+            jobParams.push_back({{ pair.first }, {vb, iter->second }});
```

Comment:
could you use `emplace_back` here?

Reply:
Yes although C++ is unable to infer the type when using emplace.  I'll revert these lines, it was unintentional.